include/deal.II/grid/grid_tools.h

```diff
@@ -2420,31 +2420,15 @@ namespace GridTools
    * article.
    */
   template <typename FaceIterator>
-  bool
+  std::optional<std::bitset<3>>
   orthogonal_equality(
-    std::bitset<3>                                               &orientation,
     const FaceIterator                                           &face1,
```

Comment:
The documentation above talks about `face_orientation`, which is already the wrong name, but I believe is what you now return. Can you update the documentation for the returned object (what it represents, and what it means if an empty optional is returned)?

Reply:
I don't understand what you're asking here - the combined orientation consists of the face orientation, face flip, and face rotation. The documentation here is correct to refer to the face orientation.

I'll add some comments about `std::optional`.

Reply:
The comment above the function referenced an argument named `face_orientation`. There was only an argument `orientation`, which you remove, so the documentation should talk about the return value rather than a named argument.

Reply:
In any case, I think this is ok now!